spec/inputs/check_boxes_input_spec.rb

```diff
@@ -358,6 +358,31 @@
 
   end
 
+  describe ':collection for a has_and_belongs_to_many association' do
+
+    before do
+      @output_buffer = ''
+      mock_everything
+
+      concat(semantic_form_for(@freds_post) do |builder|
+        concat(builder.input(:authors, as: :check_boxes, collection: Author.all))
+      end)
+    end
+
+    it 'should render checkboxes' do
+      # I'm aware these two lines test the same thing
```

Comment:
Rather than noting that you're aware, could you update the comment to discuss _why_ you're doubt-testing? What's the rationale behind it? Could you instead check that `::Author.all.size` is a meaningful number and remove the appearance of double-testing?